session/nc_creds_sessions_store_test.go

```diff
@@ -0,0 +1,650 @@
+package session_test
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/NordSecurity/nordvpn-linux/config"
+	"github.com/NordSecurity/nordvpn-linux/internal"
+	"github.com/NordSecurity/nordvpn-linux/session"
+	"github.com/NordSecurity/nordvpn-linux/test/category"
+	"github.com/NordSecurity/nordvpn-linux/test/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNCCredentialsSessionStore_Renew_NotExpired(t *testing.T) {
+	category.Set(t, category.Unit)
+
+	uid := int64(123)
+	futureTime := time.Now().UTC().Add(24 * time.Hour)
```

Comment:
Care to extract `futureTime` and `pastTime` into in-test constants? Seem to be used in here quite a few times.